first-analysis-steps/minimal-dv-job.md

```diff
@@ -133,6 +133,22 @@ to the `UserAlgorithms` list.
 The `TupleFile` attribute defines the name of the ROOT output file that DaVinci
 will store any algorithm output in, which should be our ntuple.
 
+{% callout "Being smart and efficient" %}
+`DaVinci` offers a possibility to loop only over events passing a certain requirement, rather than all events in the DST. This feature can save a lot of processing time and computing resources when running over millions of events.
```

Comment:
* "a possibility" -> "the possibility"

Is it worth explaining why can save a lot of time?

Reply:
Also: might be worth explicitly stating that using filters is _strongly_ recommended, to save yourself time and to save our collaboration's resources.

Reply:
Could you please precise what do you mean by 'Is it worth explaining why can save a lot of time?'?  You wish to add some arguments more precise than simply 'looping over events passing the requirement'?